crates/bevy_pbr/src/meshlet/gpu_scene.rs

```diff
@@ -781,13 +790,30 @@ impl FromWorld for MeshletGpuScene {
             ),
             downsample_depth_bind_group_layout: render_device.create_bind_group_layout(
                 "meshlet_downsample_depth_bind_group_layout",
-                &BindGroupLayoutEntries::sequential(
-                    ShaderStages::FRAGMENT,
+                &BindGroupLayoutEntries::sequential(ShaderStages::COMPUTE, {
+                    let write_only_r32float = || {
+                        texture_storage_2d(TextureFormat::R32Float, StorageTextureAccess::WriteOnly)
+                    };
                     (
-                        texture_2d(TextureSampleType::Float { filterable: false }),
+                        texture_depth_2d(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        texture_storage_2d(
+                            TextureFormat::R32Float,
+                            StorageTextureAccess::ReadWrite,
+                        ),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
```

Comment:
I guess this could help a bit

```suggestion
                        // view depth
                        texture_depth_2d(),
                        // mip 1
                        write_only_r32float(),
                        // mip 2
                        write_only_r32float(),
                        // mip 3
                        write_only_r32float(),
                        // mip 4
                        write_only_r32float(),
                        // mip 5
                        write_only_r32float(),
                        // mip 6
                        texture_storage_2d(
                            TextureFormat::R32Float,
                            StorageTextureAccess::ReadWrite,
                        ),
                        // mip 7
                        write_only_r32float(),
                        // mip 8
                        write_only_r32float(),
                        // mip 9
                        write_only_r32float(),
                        // mip 10
                        write_only_r32float(),
                        // mip 11
                        write_only_r32float(),
                        // mip 12
                        write_only_r32float(),
```